app/models/submission.rb

```diff
@@ -183,10 +192,7 @@ def self.get_submission_by_group_and_assignment(group_n, ass_si)
   end
 
   def make_remark_result
-    remark = create_remark_result(
-      marking_state: Result::MARKING_STATES[:unmarked],
-      submission_id: id)
-    self.save
+    remark = Result.create(marking_state: Result::MARKING_STATES[:unmarked], submission: self, remark_request_submitted_at: Time.now)
```

Comment:
Line is too long. [133/80]<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.